orca-core/src/main/java/com/netflix/spinnaker/orca/pipeline/tasks/EvaluateVariablesTask.java

```diff
@@ -36,6 +36,11 @@
   public EvaluateVariablesTask() {
   }
 
+  /**
+   * Note: Because the variables are evaluated by (pretty much all) task handlers
+   * i.e. in ExpressionAware.Stage#withMergedContext
+   * all the task needs to do is to copy the variables to the outputs
```

Comment:
A key part of this stage is the "copy" - because variables are evaluated and then saved as the non-expression version. Is there a way to make this comment clearer?

Reply:
that's what I wanted to convey, clearly that didn't work :)
How about:
```
This task copies previously evaluated expressions to the outputs map for consumption by subsequent stages.
It does not need to evaluate those variables because they would've been evaluated already by a call to e.g. ExpressionAware.Stage#withMergedContext
```

Reply:
I like this wording better but I think "it does not need to evaluate those variables" is not really true - they are evaluated in that stage, they're just not evaluated with a direct call. They're evaluated in behind the scenes magic.

Reply:
Doesn't "...because they would've been evaluated already by..." say that?

Reply:
how about:

This task copies previously evaluated expressions to the outputs map for consumption by subsequent stages.
The variables aren't evaluated here because would've been evaluated already by a call to e.g. ExpressionAware.Stage#withMergedContext

Reply:
That's great. I know it's super explicit, but hopefully it will help future debuggers.